filebeat/filebeat.full.yml

```diff
@@ -490,6 +490,19 @@ filebeat.prospectors:
 #- add_locale:
 #    format: offset
 #
+# The following example enriches each event with docker metadata, it matches
+# given fields to an existing container id and adds info from that container:
+#
+#processors:
+#- add_docker_metadata:
+#    match_fields: ["system.process.cgroup.id"]
```

Comment:
As for the example, you may want to have docker info on top of the cgroup id, so you can filter by, for instance, docker image